.github/workflows/build.yaml

```diff
@@ -3,37 +3,53 @@ name: build
 on:
   push:
     branches: [ '*' ]
-  pull_request:
```

Comment:
The build was building twice for each push so I removed this. I can put it back in if you want.